File: java/program/Shirt.java

public class Shirt extends Clothing {

    private String color, sleeveType;

    


    public Shirt(String idProduct, String name, String brand, int price, String size, String material, String gender,
            String color, String sleeveType) {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public Shirt(String color, String sleeveType) {
        this.color = color;
        this.sleeveType = sleeveType;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeveType() {
        return sleeveType;
    }

    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }

    
    
}

File: python/program/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):

    def __init__(self,id = "", name = "", brand = "", price = 0, size = "", material = "", gender = "", color = "", sleeve_type = ""):
        super().__init__(id, name, brand, price, size, material, gender)
        self.__color = color
        self.__sleeve_type = sleeve_type
    
    def getColor(self):
        return self.__color
    
    def getSleeveType(self):
        return self.__sleeve_type
    
    def setColor(self, color):
        self.__color = color

    def setSleeveType(self, sleeve_type):
        self.__sleeve_type = sleeve_type



    

    

    